tests/unit/test_pareto.py

```diff
@@ -111,3 +111,22 @@ def test_pareto_2d_bounds() -> None:
     npt.assert_allclose(
         pareto_2d.front, tf.constant([[0.1419, 0.9340], [0.1576, 0.7922], [0.4854, 0.0357]])
     )
+
+
+def test_hypervolume() -> None:
```

Comment:
the naming convention for tests for methods in trieste is usually `test_<class>_<method>` with `_<what we're testing>` at the end if there's more than one test for that method.
```suggestion
def test_pareto_hypervolume() -> None:
```